### PR TITLE
Fixed repository_compare examples

### PR DESCRIPTION
**Pull request by velvetz7**

### CODE REVIEW

File: docs/gl_objects/projects.py [hunk 1]
```diff
@@ -178,11 +178,11 @@
 result = project.repository_compare('master', 'branch1')
 
 # get the commits
-for i in commit:
-    print(result.commits)
+for commit in result.commits:
```
Comment (gpocentek) [inline docs/gl_objects/projects.py]: `result` is a dict, so you need `result['commits']` here.

File: docs/gl_objects/projects.py [hunk 2]
```diff
@@ -178,11 +178,11 @@
...
+    print(commit)
 
 # get the diffs
-for file_diff in commit.diffs:
+for file_diff in result.diffs:
```
Comment (gpocentek) [inline docs/gl_objects/projects.py]: Same here for diffs.

**Comment by velvetz7** [comment]
You are right, I fixed them.

**Comment by gpocentek** [comment]
Looks good, thank you for the quick update!

**Pull request merged**